src/services/api/actions/getListingInformationById.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Gets a listing information from the Listing Information API matching the requested concept ID(s)
+ *
+ * @param {Object} params the parameters to fetch
+ * @param {string[]} params.queryParam the concept ID(s) to match
```

Comment:
Function takes only one param, this does not match function signature. Could you correct this please.

Reply:
Fixed!